build/WFLY-16863_Std_configs_galleon_layers.adoc

```diff
@@ -0,0 +1,331 @@
+
+===== Changes to existing Galleon layers
+
+Some changes will be applied to some existing Galleon layers to reduce the configuration adjustments done in default standalone configurations.
+
+This proposal would bring the following changes:
+
+* `remoting` Galleon layer. Unset the endpoint worker `default` value. This would make the worker attribute to rely on its default value that is `default`.
+* `ejb` Galleon layer. Make dependency on `messaging-activemq` optional.
+
+
+===== Default configs defined in wildfly-ee-galleon-pack are inherited by wildfly-galleon-pack
+
+There is no re-definition of configurations in `wildfly-galleon-pack`. The configs are inherited from its dependency on `wildfly-ee-galleon-pack`.
+In order for `wildfly-galleon-pack` to include some microprofile related configuration items, the `internal-standalone-profile` 
+Galleon layer is redefined in `wildfly-galleon-pack` to include microprofile content.
+
+===== Default configs defined in wildfly-preview-feature-pack are fully redefined
+
+The wildfly-preview-galleon-pack currently fully redefine the default configurations , `internal-standalone-profile` layer 
+and `internal-standalone-full-profile` layer. 
+
+===== EE Standalone default configurations included Galleon Layers adjustments
+
+These adjustments are subject to evolve according to what changes can be done during WF28 life cycle. 
+They are the one that would be required for WF 27.
+
+* standalone.xml
+** Include `undertow-default-config` feature-group. Doesn't depend on extra JBoss Modules modules. 
+** Include `infinispan-local-server` feature-group. Doesn't depend on extra JBoss Modules modules. 
+** Unset `subsystem.ejb3` feature `default-resource-adapter-name` and `default-mdb-instance-pool` attributes.
+
+* standalone-ha.xml
+** Include `undertow-default-config` feature-group
+** Unset `subsystem.ejb3` feature `default-resource-adapter-name` and `default-mdb-instance-pool` attributes.
+
+* standalone-full.xml
+** Include `undertow-default-config` feature-group. Doesn't depend on extra JBoss Modules modules. 
+** Include `infinispan-local-server` feature-group. Doesn't depend on extra JBoss Modules modules. 
+** Include `subsystem.ejb3.service.iiop` feature. Doesn't depend on extra JBoss Modules modules. 
+
+* standalone-full-ha.xml
+** Include `undertow-default-config` feature-group. Doesn't depend on extra JBoss Modules modules. 
+** Include `messaging-activemq-ha` feature-group. Doesn't depend on extra JBoss Modules modules. 
+
+====== Full Standalone default configurations adjustments
+
+In addition to the adjustments done in wildfly-ee-galleon-pack, 
+the WF27 existing adjustments to handle microprofile are done in all default configurations:
+
+* Include `microprofile-jwt` Galleon layer
+* Include `microprofile-opentracing` Galleon layer
+* Include `microprofile-opentracing-jaeger` feature-group. Doesn't depend on extra JBoss Modules modules. 
```

Comment:
MP OT will be replaced by MP Telemetry or just OpenTelemetry (@jasondlee knows which)